src/main/pySources/graphouse_api.py

```diff
@@ -38,7 +38,14 @@ def __init__(self, path, reqkey='empty', graphouse_url='http://localhost:2005'):
         self.path = None
         self.graphouse_url = graphouse_url
 
-        if hasattr(path, '__iter__'):
+        try:
```

Comment:
This `try/catch` block looks like redundant

Reply:
Oh, sorry, that's should be removed, indeed.

Reply:
Done